demo/src/main/java/hu/flowacademy/stockmarket/config/CustomUserDetailsService.java

```diff
@@ -0,0 +1,24 @@
+package hu.flowacademy.stockmarket.config;
+
+import hu.flowacademy.stockmarket.persistance.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import javax.transaction.Transactional;
+
+@Configuration
+@Transactional
+public class CustomUserDetailsService implements UserDetailsService {
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
Legyen inkább itt is lomokos megoldás az injektálásra

Reply:
Javítva.

Reply:
Itt még azért hiányzik egy `@AllArgsConstructor`